src/models/__tests__/rc_configuration.test.ts

```diff
@@ -0,0 +1,135 @@
+  testEnvironment: {
+    ...aRemoteContentEnvironmentConfiguration,
+    testUsers: []
+  }
+};
+
+const aRetrievedRemoteContentConfigurationWithTestEnv: RetrievedRCConfiguration = {
+  ...aRemoteContentConfigurationWithTestEnv,
+  id: `${aRemoteContentConfigurationWithTestEnv.configurationId}-${"0".repeat(
+    16
+  )}` as NonEmptyString,
+  version: 0 as NonNegativeInteger,
+  _etag: "_etag",
+  _rid: "_rid",
+  _self: "_self",
+  _ts: 1
+};
+
+const aRemoteContentConfigurationWithBothEnv: RCConfiguration = {
+  ...aRemoteContentConfigurationWithNoEnv,
+  prodEnvironment: aRemoteContentEnvironmentConfiguration,
+  testEnvironment: {
+    ...aRemoteContentEnvironmentConfiguration,
+    testUsers: []
+  }
+};
+
+const aRetrievedRemoteContentConfigurationWithBothEnv: RetrievedRCConfiguration = {
+  ...aRemoteContentConfigurationWithBothEnv,
+  id: `${aRemoteContentConfigurationWithProdEnv.configurationId}-${"0".repeat(
+    16
+  )}` as NonEmptyString,
+  version: 0 as NonNegativeInteger,
+  _etag: "_etag",
+  _rid: "_rid",
+  _self: "_self",
+  _ts: 1
+};
+
+describe("remote_content", () => {
+  it("GIVEN a valid remote_content object with test environment WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRemoteContentConfigurationWithTestEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a retrieved remote_content object with test environment WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRetrievedRemoteContentConfigurationWithTestEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a valid remote_content object with prod environment WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRemoteContentConfigurationWithProdEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a retrieved remote_content object with prod environment WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRetrievedRemoteContentConfigurationWithProdEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a valid remote_content object with both environments WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRemoteContentConfigurationWithBothEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a retrieved remote_content object with both environment WHEN the object is decoded THEN the decode succeed", async () => {
+    const result = RCConfiguration.decode(
+      aRetrievedRemoteContentConfigurationWithBothEnv
+    );
+    expect(E.isRight(result)).toBeTruthy();
+  });
+
+  it("GIVEN a not valid remote_content object with no environments WHEN the object is decoded THEN the decode fail", async () => {
```

Comment:
we need to rename `remote_content` to `RCConfiguration`

Reply:
[b4a1dde](https://github.com/pagopa/io-functions-commons/pull/357/commits/b4a1ddeb3763948670cf8444f599638e68b4630a)